src/java/org/apache/cassandra/management/api/TopLevelCommand.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.management.api;
+
+import org.apache.cassandra.management.CassandraHelpCommand;
+import picocli.CommandLine;
+
+import static org.apache.cassandra.management.CassandraHelpCommand.printTopCommandUsage;
+
+@CommandLine.Command(name = "nodetool",
+    description = "Manage your Cassandra cluster",
+    subcommands = { CassandraHelpCommand.class,
+                    AbortBootstrap.class,
+                    Assassinate.class,
+                    ForceCompact.class,
+                    Compact.class })
```

Comment:
If specifying here gets messy, we can always add the subcommands in `createCommandLine()` right?

Reply:
Yes, we can specify them the same way as we do it now for the airline library, but I'd only do that in plugins/extensions, as it's better to keep commands configured statically for Cassandra main.